### DISCUSSION TITLE
Discuss Renewal of Cometa ASASTATS -> ASASTATS Staking Pool

### DISCUSSION THREAD

**Discussion by AlgoRhythMatic** [asastats/channel#348] [General]
Hello!  ASA Stats has had an ongoing ASASTATS staking pool presence at Cometa.farm over the past several months as discussed at length here:
https://github.com/asastats/channel/discussions/137

Currently, we have an active ASASTATS -> ASASTATS staking pool w/ ~$15500 TVL that  achieves an APR of ~22.8%.  To enhance stability, this staking pool has a 7-day locking mechanism where interest may only be claimed after 7 days of staking, and then every 7 days after each subsequent claim.

This pool is coming to an end in ~4 days, so in the interest of community engagement, we are seeking feedback regarding how we should proceed w/ a pool renewal.  

To get the discussion started, here are a few possibilities:
- Renew pool w/ currently designated length and allocation
    - 5,400,000 ASASTATS rewards for staking pool w/ 90 days duration with 7-days locking.
- Renew pool w/ increased length + decreased allocation (reduce rewards)
    - ??? ASASTATS rewards for staking pool w/ 120+ days duration with 7-days locking.
- Renew pool w/ decreased length and/or increased allocation (increase rewards)
    - ??? ASASTATS rewards for staking pool  w/ 30-60 days duration with 7-days locking.
- Don't renew pool and proceed with some other course of action

Due to the proximity of the pool expiration, we are hoping to come to a quick consensus on course of action over the next few days.  

Please join the discussion!

**Comment by marucat27** [comment]
If the premium tiers/use cause is not imminent, I am in favor of a lower apy and longer duration.  Once premium tiers are available, and maybe users can gain access to these tiers either by staking, LP, or some other way, I can see users trading for ALGO possibly until the use case is enabled.  The staking pool is a place for users to earn passive and I think even it's is lowered they will enjoy the fact that there is a place to stake it.  Thoughts?

**Reply by marucat27** [reply]
But if premium tiers will be available soon, I would say we can opt for same or higher apy with shorter duration.  Id say it depends on when we think the premium tiers might be ready or use case of the token.  Once the use case is enabled, people will want to hold, stake, or enter LP most likely and less likely to trade/sell for $ALGO.

**Reply by marucat27** [reply]
An example of low apy and long duration that works is $DEFLY on algofi as the use case of token is enabled currently.  I don't see much sell pressure as the use case of the token is enabled and the apy is small enough where people can stake it and feel good about earning passive apy.  It's a sustainable amount.

**Reply by ipaleka** [reply]
As we won't be ready any time soon for the development and deployment of our Permission ASC, there's a plan that we use third-party providers in the first phase. We mentioned that to Geo in the meeting.

Basically, you'd be staking your ASASTATS on for example Cometa, and the ASA Stats engine will be seeing you just like in the future it will be seeing staking in our Permission ASC. Of course, in the first period you won't be able to use all the benefits of your tier (as they aren't implemented yet), but the governance seat update will be counted from day one. 

Also, during the period when all the functionalities aren't implemented yet, you'll be getting rewards from staking as compensation - as subscribed functionalities aren't all available. In short, at the beginning of the first year of staking you'd be getting 100% of rewards (as none of the additional functionalities for subscribers are available yet), while in the last month you'd be getting no rewards (as we're planning to have all the functionalities for you tier available).

**Comment by kerrilija** [comment]
Let's try with some exact numbers. For instance, I can swap 1000 ALGO (280 USD) for approx. 2M ASASTATS. Recieveing 20,000 ASASTATS after 7 days(2.60 USD) could be tempting to a lot of people. 

I don't know how much APR % it translates to (about 100% ?). Is that too generous? Will it cause a significant price drop?

**Reply by AlgoRhythMatic** [reply]
In terms of numbers, let's consider the following:
- Current TVL of all staked ASASTATS is ~$14850 USD.  Using a mid-value swap calculation as you did above, where ~2M ASASTATS = ~1000 ALGO = ~275 USD, we arrive at an approximation that there are ~108M ASASTATS currently staked at Cometa.  Let's call this staked value of ASASTATS X.

- Next, we could take any proposed value that ASA Stats will provide as a reward (lets's call this Y) for some period of time (let's call it Z).

- In our current pool, Y = 5.4M and Z = 90 days

- Using the formula (Y/X)/Z, we can then arrive at the daily return of .000555 

- If we then multiply by 365, we can arrive at ~APR of 20%

Based on the data above, I think our goal here is to establish what we want for:

- Y - amount of ASASTATS we as a community should seek to "spend" from the project, that would be considered sustainable on an ongoing basis.

- Z - duration of the pool.  Can be more or less than current, but it seems same/more is most convenient to not have to renew as often

I think it is probably safe to consider X will remain at >108M for now, as it seems reasonable that current stakers would continue to do so in a new pool.

So my questions: 
1) is ~20% APR good/bad/other?
2) is there any reason to aim to increase or decrease this APR?
3) if we continue with another 5.4M investment (which is .005 of total ASASTATS) every 90 days, this is effectively 2% of the total 1B ASASTATS per year.  Is this sustainable?

**Reply by kerrilija** [reply]
20% APR for a non-whale-myself is not great, not terrible :D

Personally, I would like higher rewards. Those higher rewards shouldn't damage other aspects ofc. 

Decreasing rewards - I don't see any pros. People may even give up staking then, since it's not lucrative enough while being somewhat risky in case the token price drops a lot.

Increasing rewards - More people want to stake, to earn more, logically they'll want to buy more tokens. That will inevitably lead to some dumps, which may be good for someone to buy dips.

In terms of sustainability, 90 days in crypto is like a year in "RL", due to its fast everchanging nature. We can change our staking plans with every pool expiration if we want. Sure, it's additional work, but maybe it's worth it since you have more control. According to algoexplorer, 52,26% ASASTATS is in circulation. I believe those rewards are expanding the circulation?

**Reply by ipaleka** [reply]
> I believe those rewards are expanding the circulation?

Currently, the Airpool still distributes them, while the Admpool and the Rwdpool are planned to take over in the next period. So the circulation doesn't increase with them.

**Comment by BartleDooDerp** [comment]
30 day lock.
High APR
90 day length
Reason for high APR. APP just came out which sets the stage of the project for some excitement. High APR usually leads to dumping like the yieldly pool way back when but it lead to volatility. The 30 day lock may slow the dumping. LP is everything though. Perhaps tiny staking reward and High LP rewards.

**Reply by kerrilija** [reply]
Is ASASTATS/ALGO LP -> ASASTATS staking possible on cometa?

**Reply by ipaleka** [reply]
> Is ASASTATS/ALGO LP -> ASASTATS staking possible on cometa?

We had it for a month after Cometa launched on Mainnet and we had been using Aeneas ALGO too, so users had been rewarded in both ALGO and ASASTATS.

Every combination is possible, including getting only ASASTATS, but at the end of this year, Cometa will be providing Liquidity as a service functionality. We/ASA Stats will be able to provide only ASASTATS, while Cometa's users will be providing ALGO and they will be protected against impermanent loss. That being said, and as our own incentive program exists, we should probably wait for LaaS instead of running such a farm.

**Reply by AlgoRhythMatic** [reply]
> Perhaps tiny staking reward and High LP rewards.

I agree with this perspective, and suggest we are already offering the "High LP" rewards portion ourselves via the home-grown TM ASASTATS/ALGO rewards (that were just sent to our wallets today).  Which leaves us with just  the "Tiny" side of the equation :)

While I don't think ~20% is tiny by any means, my opinion is that we should offer this same value, or maybe a bit less if we extended the duration to something like 120 days.  In the 120-day use case, with the current ~108M of stake, and a re-up of 5.4M ASASTATS, we'd arrive at ~15% APR.   

Is ~15% sufficient, or should we simply renew as-is?

**Reply by marucat27** [reply]
Is it possible to offer two options?  A lower apy for a shorter lock period, and a higher apy for a longer lock period?  I believe Opulous staking pool offers something similar with two pools of different apy for different lock periods?  The user can choose which apy and lock period they want?

**Reply by RanTask** [reply]
Perhaps tiny staking reward and High LP rewards.

A fully support this approach. LP rewards are most important to ensure we maintain a sufficient liquidity pool.

**Reply by ipaleka** [reply]
Yes, LP rewards are most important, but the thing is that you get [20.000 ASASTATS](https://github.com/asastats/channel/discussions/348#discussioncomment-4188005) every 7 days for staking on Cometa those 1,000 ALGO worth of ASASTATS, while if you provide liquidity to Tinyman's ASASTATS-ALGO worth 1,000 ALGO (500 ALGO in ASASTATS) you'd get **9 times more**.

I suggest that we increase staking rewards on Cometa by at least 50% for the same 90 days (if 90 days will be our choice).

**Reply by AlgoRhythMatic** [reply]
> I suggest that we increase staking rewards on Cometa by at least 50% for the same 90 days (if 90 days will be our choice).

1.5 * 5.4 = 8.1.  If we say 8.1 for 90 days, the ~APR moves to roughly 30% (assuming consistent 108M TVL).  This sounds reasonable and sustainable to me, but may be lower than what @kerrilija is targeting.

I personally don’t see us increasing 200-400% to get ~APR from 20% now to  say 60-100% for an ongoing basis…it would not be sustainable over time.

Edit: or at least would not be sustainable until some income was coming in.

**Reply by ipaleka** [reply]
Yes, I also don't want it that much higher. Mainly because of this what @RanTask says, but that's not the only reason.

Our Tinyman's ASASTATS-ALGO LP has been growing, at a slow pace tbh, but it's still growing of sorts. Logic says its growth will be at the expense of other stakings and that's only Cometa at this moment.

But the main reason is that we will have that counterpart to our Permission ASC and that program will surely make some people withdraw from that ASASTATS Cometa staking. And that ultimately means this Cometa's staking program will get higher APR in the second, third month than your/our calculations.

**Reply by kerrilija** [reply]
That's why I've asked if it is too generous. Liquidity is definitely higher priority than staking rewards. Thank you all for your answers! It is obviously a different perspective about rewards if you are to give them or to recieve them. I suggested those values by looking at them as a regular Cometa "user", what would attract me. Apparently, it's not sustainable, especially with those nice LP rewards. That being said, the raise to 30% APR is great, knowing that we have both options for earning more ASASTATS (LP, cometa staking).

**Comment by AlgoRhythMatic** [comment]
So - is the current proposal an increase to 8.1M rewards for a 90-day period in a single 7-day locking stake pool - for an ~ target APR of 30% (subject to decrease as participation increases)?

**Reply by marucat27** [reply]
That sounds good to me 👍

**Reply by ipaleka** [reply]
Yes, sounds good to me too. :) If nobody complains until tomorrow, @AlgoRhythMatic you may send them a similar message to TG group and I'll send those 8.1m to their escrow afterward.

**Reply by RanTask** [reply]
Sounds good to me, too!

**Reply by AlgoRhythMatic** [reply]
I think we have now a quorum, so I've posted the following in a Telegram message to Cometa team:

```
Hello @Geo2856 and @nikitacometa!  We opened a discussion regarding continuation of our ASASTATS staking pool here in GitHub:
https://github.com/asastats/channel/discussions/348

Our conclusion:
After our current staking pool expires in ~2 days, we'd like to start a new pool with the following:
- 7-day locking
- 90-day duration
- Stake ASASTATS -> ASASTATS rewards
- Funded with 8.1M ASASTATS (to be provided by ipaleka hereafter confirmation).

Are there any other details needed?
```
